.storybook/config.js

```diff
@@ -1,6 +1,14 @@
 import { configure, setAddon } from '@storybook/react'
 import { setOptions } from '@storybook/addon-options'
 import chaptersAddon from 'react-storybook-addon-chapters'
+import { addParameters } from '@storybook/react'
+
+addParameters({
```

Comment:
Could we use [@storybook/addon-knobs](https://www.npmjs.com/package/@storybook/addon-knobs) for configuration for consistency with the existing setup?

Reply:
Knobs are component specific. What we have here is a global theme definition, that's why the switcher is on top if that makes sense.